### DISCUSSION TITLE
What is the difference between flopy and MODFLOW API?

### DISCUSSION THREAD

**Discussion by qiyang77** [modflowpy/flopy#1676] [General]
The MODFLOW API is described here [https://www.usgs.gov/publications/modflow-application-programming-interface-simulationcontrol-and-software](url). But I have no idea what's the difference between them.

**Comment by jlarsen-usgs** [comment]
@qiyang77 

FloPy is a package is package for developing, editing, reading, and writing modflow input files as well as performing data visualization and output analysis.

The MODFLOW API is the shared fortran library to MODFLOW6. This shared library allows users to interface with MODFLOW while it's running, change input values for stresses, and even develop their own MODFLOW packages in another programming language. There is a python interface to the MODFLOW API, and extensions are currently being reviewed to create a more intuitive UI. The MODFLOW API python interface is located at: [modflowapi](https://github.com/MODFLOW-USGS/modflowapi).
